pymumble_py3/crypto.py

```diff
@@ -162,6 +162,9 @@ def decrypt(self, source: bytes, len_plain: int) -> bytes:
         if len(source) < 4:
             raise DecryptFailedException('Source <4 bytes long!')
 
+        if not len_plain:
+            len_plain = len(source) - 4  # The entire packet length, minus the header
```

Comment:
I found out that it was best to just remove the `len_plain` as a parameter to `decrypt` and just use this line. The problem that I hit was that if `len_plain` was passed in, it contained the length including the header and that caused issues with decrypting later on. The call to `decrypt` basically just passes `len_plain` as `len(source)` anyway.

Reply:
This should be solved now